content/blog/2020/05/2020-05-06-docker-agent-image-renaming.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Renaming of official Docker agent images"
+tags:
+- announcement
+- docker
+- platform-sig
+author: oleg_nenashev
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We would like to announce renaming of the official Docker images for Jenkins agents.
+This article provides information about the new official names, upgrade procedure, and the support policy for the old images.
+This renaming does not have any immediate impact on Jenkins users.
+
+image:/images/docker/dockerJenkins.png[Jenkins and Docker, role=center, float=right, height=224]
+
+== New image names
+
+* link:https://hub.docker.com/repository/docker/jenkins/agent[jenkins/agent] is a new name of the old link:https://hub.docker.com/repository/docker/jenkins/slave[jenkins/slave] image,
+  starting from link:https://github.com/jenkinsci/docker-agent/releases/tag/4.3-2[4.3-2]
+* link:https://hub.docker.com/repository/docker/jenkins/inbound-agent[jenkins/inbound-agent] is a new name of the  link:https://hub.docker.com/repository/docker/jenkins/jnlp-slave[jenkins/jnlp-slave] image,
+  starting from link:https://github.com/jenkinsci/docker-inbound-agent/releases/tag/4.3-2[4.3-2]
+* link:https://hub.docker.com/repository/docker/jenkins/agent[jenkins/agent] is a new name of the old link:https://hub.docker.com/repository/docker/jenkins/ssh-slave[jenkins/ssh-slave] image,
+  starting from link:https://github.com/jenkinsci/docker-ssh-agent/releases/tag/2.0.0[2.0.0]
+
+See the upgrade guidelines below.
+
+== Why?
+
+The "slave" term is widely considered inappropriate in open source communities.
+It has been officially deprecated in Jenkins 2.0 in 2016, but there are remaining usages in some Jenkins components.
+The jira:JENKINS-42816[Slave to Agent renaming leftovers] EPIC tracks cleanup of such usages.
+Official Docker agent images were a glaring case, it was not easy to fix that with the previous versions of the image release Pipelines on DockerHub.
+It great to have the image naming issue finally fixed by this update.
```

Comment:
There is a slight grammatical error in line 36. It should be : 
It is great to have the image naming issue finally fixed by this update

Reply:
```suggestion
It is great to have the image naming issue finally fixed by this update.
```